src/vega.schema.ts

```diff
@@ -148,6 +148,15 @@ export interface VgFormulaTransform {
   expr: string;
 }
 
+export interface VgAggregateTransform {
+  type: 'aggregate';
+  groupby: VgFieldRef[];
+  fields: VgFieldRef[];
+  ops: string[];
+  as?: string[];
+  drop?: boolean;
```

Comment:
Side Note for future:

This property is interesting.  We should think about a way to specify this.
For `summarize` transform in `transform`, we can just add `drop` property.

For channelDef's inline transform, this is trickier though.
(We could add this to config, but should also consider non-config approach.